generators/newapp/templates/templates/application.html.tmpl

```diff
@@ -24,5 +24,10 @@
     </div>
 
     <%= javascriptTag("application.js") %>
+
+    <%= if( env == "development") { %>
+    <!-- For buffalo to do live js/css reload -->
+    <script src="http://localhost:35729/livereload.js"></script>
```

Comment:
What if we have `javascriptTag` automatically generate `<script src="http://localhost:35729/livereload.js"> </script>` too?

Reply:
it seems the plugin provides a way to inject this script in the header without putting it here, so i removed this portion of the changes :)